src/components/veNft/formControls/StakingFormActionButton.tsx

```diff
@@ -31,7 +31,12 @@ const StakingFormActionButton = ({
     }
     if (!hasAdequateApproval) {
       return (
-        <Button block style={{ whiteSpace: 'pre' }} onClick={approve}>
+        <Button
```

Comment:
**suggestion** think we can add `type="primary"` here

Reply:
addressed in downstream branch

Reply:
done